crates/eth-rpc/src/api/eth_api.rs

```diff
@@ -9,6 +9,11 @@ use reth_rpc_types::{
 #[rpc(server, namespace = "eth")]
 #[async_trait]
 pub trait EthApi {
+    /// Returns true if RPC_URL is online
+    #[method(name = "health")]
```

Comment:
I don't think this method should be in the EthApi, 
The EthApi is a documented and quite mature specification:
https://ethereum.github.io/execution-apis/api-documentation/

We should implement it in a custom Api.
Suggestion:
- Delete this function in EthApi (eth_api.rs)
- Delete the implementation of the trait in server/eth_rpc.rs


This will still make net_health reachable, but eth_health, not